squad_client/manage.py

```diff
@@ -20,6 +29,15 @@ def main():
         parser.print_help()
         return -1
 
+    squad_host = args.squad_host or os.getenv('SQUAD_HOST')
+    if squad_host:
+        try:
+            squad_token = args.squad_token or os.getenv('SQUAD_TOKEN')
```

Comment:
I would check if squad_token isn't empty.

Reply:
just for the record, there's already a check for empty token here: https://github.com/Linaro/squad-client/blob/master/squad_client/core/api.py#L45